src/region.js

```diff
@@ -84,9 +84,9 @@ const Region = MarionetteObject.extend({
     }
   },
 
-  _attachView(view) {
+  _attachView(view, {replaceElement} = {}) {
     const shouldTriggerAttach = !view._isAttached && isNodeAttached(this.el);
-    const shouldReplaceEl = !!this.getOption('replaceElement');
+    const shouldReplaceEl = !!replaceElement || !!this.getOption('replaceElement');
```

Comment:
actually.. doesn't this have to be something like `replaceElement !== false`  In this case if `replaceElement` is explicitly false, but `this.replaceElement` is true, the option won't override the class?